add an answer to the Zaraz FAQ (#13069)
cloudflare/cloudflare-docs: content/zaraz/faq.md

{{</faq-answer>}}
{{</faq-item>}}

{{<faq-item>}}

> {{<faq-question level=4 text="How can I attach User Properties to my events?" >}}

Reviewer: ```suggestion
{{<faq-question level=4 text="How do I attach a User Property to my events?" >}}
```

{{<faq-answer>}}

> You can add User Properties by choosing "Add field" in your Google Analyitcs 4 action, and selecting "Add custom field...". Give the field a name that starts with `up.`, for example, `up.name`. This will make Zaraz send this field as a User Property and not as an Event Property.

Reviewer: ```suggestion
In your Google Analytics 4 action, select **Add field** > **Add custom field...** and enter a field name that starts with `up.` — for example, `up.name`. This will make Zaraz send the field as a User Property and not as an Event Property.
```
